src/server/pps/pretty/pretty.go

```diff
@@ -80,10 +81,9 @@ func PrintJobSetInfo(w io.Writer, jobSetInfo *ppsclient.JobSetInfo, fullTimestam
 	var created *types.Timestamp
 	var modified *types.Timestamp
 	for _, job := range jobSetInfo.Jobs {
-		switch job.State {
-		case ppsclient.JobState_JOB_SUCCESS:
+		if job.State == ppsclient.JobState_JOB_SUCCESS {
 			success++
-		case ppsclient.JobState_JOB_FAILURE:
+		} else if ppsutil.IsTerminal(job.State) {
```

Comment:
This wasn't handling the `KILLED` state correctly, so tried to future-proof it a bit for arbitrary failed states.